src/UI/Internal/NList.elm

```diff
@@ -1,41 +0,0 @@
-module UI.Internal.NList exposing (..)
```

Comment:
I should have put this in an external package, but I agree in deleting it :smile: